--- XMP-Toolkit-SDK-4.4.2/source/XMPFiles/FormatSupport/IPTC_Support.hpp.orig	2014-09-02 23:05:24.714933593 +0100
+++ XMP-Toolkit-SDK-4.4.2/source/XMPFiles/FormatSupport/IPTC_Support.hpp	2014-09-02 23:05:35.081379730 +0100
@@ -18,6 +18,8 @@
 #include "XMPFiles_Impl.hpp"
 #include "EndianUtils.hpp"
 
+#include <cstdlib>
+
 // =================================================================================================
 /// \file IPTC_Support.hpp
 /// \brief XMPFiles support for IPTC (IIM) DataSets.
